client/static/ui_dropdowns/requesttype.json

```diff
@@ -6,7 +6,9 @@
 		"CLASS_TYPE_CD": "CORP",
 		"LEGACY_CD": "CR",
 		"ADDITIONAL_INFO": "",
-		"SOURCE_APPLICATION": "COLIN"
+		"SOURCE_APPLICATION": "COLIN",
```

Comment:
Fix original indents above/below (or mimic them for new code).

Reply:
Yea, i'm not sure why it displays like that in the diff. If you view file it displays the indents correctly. 
Is that what you mean?

Reply:
Yes, that's what I'm referring to.

Probably some of that whitespace is tabs and some is spaces. (Some editors auto-replace tabs with spaces.) You might be able to replace all tabs with 2 spaces (or whatever) to fix this.

This is P3 not P1. Nice to have. Only do if time allows :)

Reply:
Yea i'll look at it for sure, mostly because it's confusing and appears normal in my editor haha

Reply:
haha ok, ready for another look. Seems like it mostly some residual weirdness. Hopefully that cleaned it up!

Reply:
Yes, all cleaned up, thanks!